clients/spring-boot-starter-camunda-sdk/src/main/java/io/camunda/zeebe/spring/client/configuration/ZeebeClientConfigurationImpl.java

```diff
@@ -39,6 +39,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.ScheduledExecutorService;
+import org.apache.hc.client5.http.async.AsyncExecChainHandler;
```

Comment:
🔧 We need to add the `org.apache.httpcomponents.client5:httpclient5` dependency in the Spring SDK now as well since we use a class from it directly.